README.rdoc

```diff
@@ -47,10 +47,23 @@ useful. If there is another library which handles it, we list it here.
 
 == Develop
 
+* Clone the repository using
+  git clone https://github.com/ruby-ldap/ruby-net-ldap
```

Comment:
Need line breaks after `* Clone the repository using`  to ensure the code is formatted, not stuck to the end of the line.

You can preview it with Github's `view file` option: https://github.com/ruby-ldap/ruby-net-ldap/blob/b1871a0af286f941ea13769e03a9500110d0a76d/README.rdoc

Same goes for the below

Reply:
Thank you. formatting done :)

Reply:
![image](https://user-images.githubusercontent.com/24943868/81967682-ed9e2a00-95e0-11ea-948e-5e8e1d46d26b.png)

If you remove the bullets, it'll match the existing code blocks

```
To install all the dependencies for local development:

  gem install bundler # if you don't have it
  bundle install

This task will run the test suite and the
```